spinalcordtoolbox/image.py

```diff
@@ -402,11 +402,11 @@ def loadFromPath(self, path, verbose):
         :return:
         """
 
-        im_file = nib.load(path, mmap=(not sys.platform.startswith('win32')))
+        self.absolutepath = os.path.abspath(path)
+        im_file = nib.load(self.absolutepath, mmap=(not sys.platform.startswith('win32')))
         self.affine = im_file.affine.copy()
         self.data = im_file.get_data()
         self.hdr = im_file.header.copy()
-        self.absolutepath = path
```

Comment:
This change addresses a bug that was found by [dedb176](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3770/commits/dedb1763a2e6b83b13ee073e05aa66a842287090), but it's possible we might want to merge this commit in a separate PR?